From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@gmail.com>
Date: Thu, 17 Oct 2024 06:10:07 +0300
Subject: [PATCH] gen_build_prop: Set ro.build.fingerprint

Change-Id: I0e40b6f8154f8bc9a7b8450ecb4af4c44ab3d46d

diff --git a/scripts/gen_build_prop.py b/scripts/gen_build_prop.py
index 5549131257f35f3500def1f7176afbdefbd3b976..1b16ff948c739ddab678df905b6a4dac7da3ca0c 100644
--- a/scripts/gen_build_prop.py
+++ b/scripts/gen_build_prop.py
@@ -197,6 +197,8 @@ def generate_build_info(args):
   config = args.config
   build_flags = config["BuildFlags"]
 
+  print(f"ro.build.fingerprint?={config['BuildFingerprint']}")
+
   # The ro.build.id will be set dynamically by init, by appending the unique vbmeta digest.
   if config["BoardUseVbmetaDigestInFingerprint"]:
     print(f"ro.build.legacy.id={config['BuildId']}")
